fc-community-extension/src/contentModifiers.js

```diff
@@ -153,27 +161,33 @@ const makeFactchainHtmlNote = (note) => {
         <span class="css-901oao css-16my406 r-poiln3 r-bcqeeo r-qvutc0">${note.content}</span>
       </span>
   `;
-}
 
-const addNote = async (mainArticle, note) => {
-  let htmlNote = makeFactchainHtmlNote(note);
   if (!note.finalRating) {
-    const provider = await createFactchainProvider();
-    const userAddress = await provider.getAddress();
-    if (!userAddress || userAddress.toLowerCase() !== note.creatorAddress.toLowerCase()) {
-      htmlNote += `<div>${rateItHtml}</div>`
+    if (!userAddress || author !== "You") {
+      noteHTML += `<div>${rateItHtml}</div>`
     }
   }
+  return noteHTML;
+}
+
+const addNote = async (mainArticle, note) => {
+  const provider = await createFactchainProvider();
+  const userAddress = await provider.getAddress();
+  let htmlNote = makeFactchainHtmlNote(note, userAddress);
   const tempDiv = mainArticle.children[0].children[0]
   const afterThisDiv = [].slice.call(tempDiv.children[tempDiv.children.length - 1].children).find(e => e.innerHTML === "");
   afterThisDiv.insertAdjacentHTML("afterend", htmlNote);
+  const rateNoteButton = mainArticle.querySelector("#rateNoteButton")
+
+  if (rateNoteButton) {
```

Comment:
you've found the `rateNoteButton` already, so inside the if's body, no need to do `mainArticle.querySelector("#rateNoteButton")` again.